src/meetings/meetingListData.js

```diff
@@ -1022,6 +1022,18 @@ export default {
                 notes: 'Agape Outpost Church'
             },
         },
+        {
+            time: '7:00 pm',
+            name: 'Miracles on the Fax',
+            format: ['SP', '90 min', 'Rotating Format'],
```

Comment:
I have been putting the time at the end to be consistent across all meetings. At some point I think we can change the schema a bit. We will be using the BMLT which has the meeting length as a property in the object we will fetch from their API.

Reply:
This will enable us to put the length at the end programmatically. :)